File: server/server-rest/src/main/java/uk/co/stevewellington/shoppinglist/MyGuiceServletConfig.java
package uk.co.stevewellington.shoppinglist;

import com.google.inject.Guice;
import com.google.inject.Injector;
import com.google.inject.servlet.GuiceServletContextListener;

import static com.google.inject.Guice.createInjector;

public class MyGuiceServletConfig extends GuiceServletContextListener {
    @Override protected Injector getInjector() {
        return createInjector(new ApplicationServletModule());
    }
}

File: server/server-rest/src/main/scala/uk/co/stevewellington/shoppinglist/MyGuiceServletConfig.scala
package uk.co.stevewellington.shoppinglist

import com.google.inject.Guice.createInjector
import com.google.inject.servlet.GuiceServletContextListener

//noinspection JavaAccessorMethodOverridenAsEmptyParen
class MyGuiceServletConfig extends GuiceServletContextListener {

  override def getInjector() = createInjector(new ApplicationServletModule)

}
